internal/acceptance/ssh_key_helpers.go

```diff
@@ -34,3 +36,29 @@ func TestAccCheckMetalSSHKeyExists(n string, key *packngo.SSHKey) resource.TestC
 		return nil
 	}
 }
+
+func TestAccCheckMetalGOSSHKeyExists(n string, key *metalv1.SSHKey) resource.TestCheckFunc {
```

Comment:
Rather than add a new function here, lets update the existing one to use equinix-sdk-go.  These check functions are only used in tests, so we can change them a bit more freely.